packages/modeling/src/maths/mat4/invert.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Creates a inverse copy of the given matrix.
+ * @author Julian Lloyd
+ * code from https://github.com/jlmakes/rematrix/blob/master/src/index.js
+ *
+ * @param {mat4} m - matrix to invert
+ * @returns {mat4} out
+ * @alias module:modeling/maths/mat4.inverse
+ */
```

Comment:
It would be best if the function name matched the file name. :)

Reply:
no need to say sorry 😺 I intended to rename both. I just forgot to rename function inside.
decided to name it invert  to match gl-matrix

Nice catch, thanks.